src/main/kotlin/exercise3/task1/BalancedBrackets.kt

```diff
@@ -24,9 +26,32 @@ package exercise3.task1
  *
  */
 
-
 internal fun isExpressionBalanced(expression: String): Boolean {
-    TODO("Implement me!!!")
+    val stack = Stack<Char>()
```

Comment:
Well done! Can you try implementing this functional way for a practice?

Reply:
I'll try